benchmarks/benchmarks/api/bench_builtin_update_all.py

```diff
@@ -7,13 +7,12 @@
 from .. import utils
 
 
-@utils.benchmark('time', timeout=7200)
-@utils.parametrize('graph_name', ['cora', 'reddit'])
-@utils.parametrize('format', ['coo', 'csr'])
-@utils.parametrize_cpu('feat_size', [8, 128, 512])
-@utils.parametrize_gpu('feat_size', [8, 32, 256])
+@utils.benchmark('time', timeout=600)
+@utils.parametrize('graph_name', ['ogbn-arxiv', 'reddit', 'ogbn-proteins'])
+@utils.parametrize('format', ['csc'])
+@utils.parametrize('feat_size', [4, 32, 256])
 @utils.parametrize('msg_type', ['copy_u', 'u_mul_e'])
-@utils.parametrize('reduce_type', ['sum', 'mean', 'max'])
```

Comment:
`mean` should be there. It is quite important and the underlying implementation depends on two kernels (in degree and spmm) not one.